bindings/nodejs/test/client/outputBuilders.spec.ts

```diff
@@ -0,0 +1,168 @@
+        },
+    ],
+    localPow: true,
+});
+
+const secretManager = {
+    Mnemonic:
+        'endorse answer radar about source reunion marriage tag sausage weekend frost daring base attack because joke dream slender leisure group reason prepare broken river',
+};
+
+describe.skip('Output builder methods', () => {
+    it('builds a basic output', async () => {
+        const addresses = await client.generateAddresses(secretManager, {
+            range: {
+                start: 0,
+                end: 1,
+            },
+        });
+
+        const hexAddress = await client.bech32ToHex(addresses[0]);
+
+        // most simple basic output
+        const basicOutput = await client.buildBasicOutput({
+            amount: '1000000',
+            unlockConditions: [
+                {
+                    type: 0,
+                    address: {
+                        type: 0,
+                        pubKeyHash: hexAddress,
+                    },
+                },
+            ],
+        });
+
+        expect(basicOutput).toBeDefined();
+    });
+
+    it('builds an alias output', async () => {
+        const addresses = await client.generateAddresses(secretManager, {
+            range: {
+                start: 0,
+                end: 1,
+            },
+        });
+
+        const hexAddress = await client.bech32ToHex(addresses[0]);
+
+        // most simple alias output
+        const aliasOutput = await client.buildAliasOutput({
+            aliasId: hexAddress,
+            unlockConditions: [
+                {
+                    type: 4,
+                    address: {
+                        type: 0,
+                        pubKeyHash: hexAddress,
+                    },
+                },
+                {
+                    type: 5,
+                    address: {
+                        type: 0,
+                        pubKeyHash: hexAddress,
+                    },
+                },
+            ],
+        });
+
+        expect(aliasOutput).toBeDefined();
+    });
+
+    it('builds a foundry output', async () => {
+        const addresses = await client.generateAddresses(secretManager, {
+            range: {
+                start: 0,
+                end: 1,
+            },
+        });
+
+        const hexAddress = await client.bech32ToHex(addresses[0]);
+
+        // most simple alias output
+        const aliasOutput = await client.buildAliasOutput({
+            aliasId: hexAddress,
+            unlockConditions: [
+                {
+                    type: 4,
+                    address: {
+                        type: 0,
+                        pubKeyHash: hexAddress,
+                    },
+                },
+                {
+                    type: 5,
+                    address: {
+                        type: 0,
+                        pubKeyHash: hexAddress,
+                    },
+                },
+            ],
+        });
+
+        const aliasId = aliasOutput.aliasId;
```

Comment:
The aliasId isn't an Ed25519 address, it can be used as alias address, but is the first time just `0x0000000000000000000000000000000000000000000000000000000000000000` and later the hash of the OutputId from the initial AliasOutput
So maybe just define the aliasId as `0xa5c28d5baa951de05e375fb19134ea51a918f03acc2d0cee011a42b298d3effa' (that's one example aliasId)